### PR TITLE
Don't print stack trace when select queries complete exceptionally

### PR DESCRIPTION
**Pull request by jwilliams-ocient**
This was something I added to aid with debugging and was not intended to be merged with #81 !

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -20,4 +20,15 @@
...
+		<filter>
+			<id>1606839504837</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
```
Comment (leogreca) [inline .project:33]: What is that for?

Comment (jwilliams-ocient) [reply .project:33]: Wow that’s what I get for using `git add -u`. I should add this to my .gitignore

Comment (jwilliams-ocient) [reply .project:33]: Will fix after dinner

**Review by tli-ocient — APPROVED** [review]
Just getting rid of stack trace print

**Review by leogreca — APPROVED** [review]

**Pull request merged**